src/OrchardCore/OrchardCore.DisplayManagement/Razor/RazorPage.cs

```diff
@@ -100,11 +106,12 @@ public Task<IHtmlContent> DisplayAsync(IShape shape)
     /// <param name="clearAlternates">A value indicating whether to clear the shape's alternates before rendering.  <see langword="true"/> to clear
     /// alternates; otherwise, <see langword="false"/>. Defaults to <see langword="true"/>.</param>
     /// <returns>A task that represents the asynchronous operation. The task result contains the rendered HTML content.</returns>
-    public Task<IHtmlContent> DisplayAsAsync(IShape shape, string shapeType, bool? clearAlternates = null)
```

Comment:
I think we should keep this as-is. I've replied here to you as well: https://github.com/OrchardCMS/OrchardCore/commit/6f22177c30f38a78e17cee5dc45e18015ac23bef#r168663688